### PR TITLE
Added UTs for GetAllTribes, GetTotalTribes & GetListedTribes

### PR DESCRIPTION
**Pull request by hkarani**
## Issue ticket number and link
Issue number 1527
https://github.com/stakwork/sphinx-tribes/issues/1527

## Type of change
Added test to tribes.go for `GetAllTribes`, `GetListedTribes`, `GetTotalTribes`

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [x] I have performed a self-review of my code

### CODE REVIEW

File: handlers/tribes_test.go [hunk 1]
```diff
@@ -471,3 +473,125 @@ func TestGetTribeByUniqueName(t *testing.T) {
...
+		handler := http.HandlerFunc(tHandler.GetListedTribes)
+		expectedTribes := []db.Tribe{
+			{UUID: "1", Name: "Tribe 1", Tags: pq.StringArray{"tag1", "tag2", "tag3"}},
+			{UUID: "2", Name: "Tribe 2", Tags: pq.StringArray{"tag4", "tag5"}},
+			{UUID: "3", Name: "Tribe 3", Tags: pq.StringArray{"tag6", "tag7", "tag8"}},
+		}
+		rctx := chi.NewRouteContext()
+		tagVals := pq.StringArray{"tag1", "tag4", "tag7"}
+		tags := strings.Join(tagVals, ",")
+		rctx.URLParams.Add("tags", tags)
```
Comment (elraphty) [inline handlers/tribes_test.go:553]: The tags should be in URL query not Params

File: handlers/tribes_test.go [hunk 2]
```diff
@@ -471,3 +473,125 @@ func TestGetTribeByUniqueName(t *testing.T) {
...
+		handler := http.HandlerFunc(tHandler.GetListedTribes)
+		expectedTribes := []db.Tribe{
+			{UUID: "1", Name: "Tribe 1", Tags: pq.StringArray{"tag1", "tag2", "tag3"}},
+			{UUID: "2", Name: "Tribe 2", Tags: pq.StringArray{"tag4", "tag5"}},
+			{UUID: "3", Name: "Tribe 3", Tags: pq.StringArray{"tag6", "tag7", "tag8"}},
+		}
+		rctx := chi.NewRouteContext()
+		tagVals := pq.StringArray{}
+		tags := strings.Join(tagVals, ",")
+		rctx.URLParams.Add("tags", tags)
```
Comment (elraphty) [inline handlers/tribes_test.go:580]: The tags should be in URLQuery, not URLParams

File: routes/tribes.go [hunk 3]
```diff
@@ -10,12 +10,13 @@ func TribeRoutes() chi.Router {
 	r := chi.NewRouter()
 	tribeHandlers := handlers.NewTribeHandler(db.DB)
 	r.Group(func(r chi.Router) {
-		r.Get("/", handlers.GetListedTribes)
+		r.Get("/{tags}", tribeHandlers.GetListedTribes)
```
Comment (elraphty) [inline routes/tribes.go:13]: Why change the URL and add tags params to it, the tags come come the URL query.

**Review by elraphty — CHANGES_REQUESTED** [review]

**Comment by hkarani** [comment]
Created another pull request to fix this.

**Pull request closed without merging**